src/uu/rm/src/rm.rs

```diff
@@ -611,3 +679,17 @@ fn is_symlink_dir(metadata: &Metadata) -> bool {
     metadata.file_type().is_symlink()
         && ((metadata.file_attributes() & FILE_ATTRIBUTE_DIRECTORY) != 0)
 }
+
+mod tests {
+
```

Comment:
```suggestion
    use super::*;
```

This way, you don't have to `use crate::foo_bar;` everything in each test again and again.